dev/com.ibm.ws.jsonb_fat/publish/servers/com.ibm.ws.jsonb.container.fat/server.xml

```diff
@@ -13,6 +13,7 @@
       <feature>componenttest-1.0</feature>
       <feature>jsonbContainer-1.0</feature>
       <feature>servlet-4.0</feature>
+      <feature>cdi-2.0</feature>
```

Comment:
After adding this, do we still have coverage that jsonb features can be used without cdi feature?

Reply:
only in-app JSON-B coverage.  Didn't seem like this code path was drastic enough to warrant another server start/stop in this bucket.